backend/docs/multi_pathogen.md

```diff
@@ -192,20 +199,39 @@ Multi-segments:
     "original_data": {
         "metadata": {...},
         "unalignedNucleotideSequences": {
-            "segment1": "ACTG",
-            "segment2": "GTCA"
+            "key1_myFirstSegment": "ACTG",
+            "key1_mySecondSegment": "GTCA"
         }
     }
 }
 ```
 
+### Problems that need to be solved:
+ 
+The backend compresses the unaligned nucleotide sequences before storing them.
+It uses the reference sequence as dictionary to achieve a good compression ratio.
+This is possible because it knows which segment a sequence is supposed to be.
+
+This will not be possible anymore.
+For single-segment organisms, the backend can still take the only segment as the dictionary.
+
+When there are multiple segments:
+* What should we use as the dictionary? Concatenation of all segments?
+* How do we make sure that after rolling out these changes, 
+  the unaligned nucleotide sequences that are already in the DB can still be decompressed (e.g. CCHF)?
+  This probably requires a non-trivial database migration.
+  The reference sequences are not store in the DB, so the migration cannot be a simple SQL script.
+
 ## preprocessing input
 
 Similar to what is stored in the DB.
 
 ## Preprocessing
 
-TODO: Minimizer index, etc?
+The preprocessing pipeline will need to assign each sequence to a segment, if there are multiple segments
+(including validation and useful error messages for the submitter).
+
+TODO: details how to do that - minimizer index, etc.?
```

Comment:
Higher level: 
- receive a group of metadata and sequences
- assign suborganism using nextclade sort (will require a minimizer index and potentially a map from match name to suborganism
- once suborganism is assigned run nextclade using the correct nextclade dataset - must be configured per organism 
- must validate that: input does not have multiple segments for one sample - for EV case could also add a config option to allow or disallow co-infections
- as currently, either have hard or soft enforcement of all or any of segments aligning

Reply:
Would you mind adding this yourself to that section? You have a better overview what to write there and I don't claim any ownership on this PR.